{"status":"OK","subtests":[{"name":"error event is weird (return true cancels; many args) on WorkerGlobalScope, with a synthetic ErrorEvent","status":"FAIL"},{"name":"error event has the right 5 args on WorkerGlobalScope, with a synthetic ErrorEvent","status":"FAIL"}]}